index.qmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Preliminary Comparison of RSE and DS Competences"
+title: "Preliminary Comparison of Research Software Engineering and Data Science Competences"
```

Comment:
I think we should use the expanded versions of these acronyms in the title since many people may not be familiar with them.

Reply:
Agreed!